lib/test/python/test_nc_checker_call.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""
+unittest for lib/python/util.py netcdf checker call function
+
+author : besnard, laurent
+"""
+
+from util import pass_netcdf_checker
+import os
+import unittest
+
+class TestPassChecker(unittest.TestCase):
+
+    def setUp(self):
+        self.nc_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'test.nc')
+
+    def test_pass_netcdf_checker(self):
+        self.assertTrue(pass_netcdf_checker(self.nc_path, ['cf:1.6']))
+        self.assertTrue(pass_netcdf_checker(self.nc_path, ['imos:1.3']))
```

Comment:
It's already done, i.e. what you have there should work. For now we only have 'imos:1.3', but eventually we'll have 'imos:1.4' as well. 'imos' and 'imos:latest will default to the latest version of the conventions.
(Run `compliance-checker -l` to see all available check suites.)